components/docs-chef-io/data/automate/cli_chef_automate/commands/chef-automate_config_patch.yaml

```diff
@@ -2,25 +2,29 @@ name: chef-automate config patch
 synopsis: patch the Chef Automate configuration
 usage: chef-automate config patch path/to/config.toml [flags]
 description: |
-  Apply a partial Chef Automate configuration to the deployment. It will take the partial configuration, merge it with the existing configuration, and apply and required changes.
+    Apply a partial Chef Automate configuration to the deployment. It will take the partial configuration, merge it with the existing configuration, and apply and required changes.
```

Comment:
these change are not required, it should be auto generated not be changed manaually